### DISCUSSION TITLE
Specifying image size in percentages

### DISCUSSION THREAD

**Discussion by 4r73m190r0s** [eclipse-birt/birt#1388] [Q&A]
I added an image inside grid cell, that has specified height of 250px. I've set image width to 100%, and it stretches to fit the container's width as expected. But, when I set image height to 100%, it doesn't fill the container's height, but its height gets very small, which is very unexpected and I have no idea why. And suggestions?

**Comment by speckyspooky** [comment]
The percentage value is the size of the image and the support to recognize the environment around the image according to row-height or grid-(width & height) isn't recognized in all situations and also the emitters of HTML, PDF and DOCX works different.
So the best way on BIRT-images is to use a fixed size instead of percentage or you calculate the image size dynamically via script.

**Reply by speckyspooky** [reply]
Mayba one alternative would be to use a background image for the grid-cell because the you can set the image to contain exactly into the cell/container. The handling of the background images was improved with the latest changes in the current built version 4.14.
